lib/node_modules/@stdlib/stats/base/meanpn/README.md

```diff
@@ -112,7 +112,7 @@ var v = meanpn.ndarray( N, x, 1, 0 );
 
 The function has the following additional parameters:
 
--   **offset**: starting index for `x`.
+-   **offsetX**: starting index for `x`.
 
 While [`typed array`][mdn-typed-array] views mandate a view offset based on the underlying `buffer`, the `offset` parameter supports indexing semantics based on a starting index. For example, to calculate the [arithmetic mean][arithmetic-mean] for every other value in `x` starting from the second value
 
```

Comment:
See other similar packages, but we also tend to add a note concerning accessor protocol support below.